test/fixtures/blocked-paths.txt

```diff
@@ -184,4 +184,505 @@
+/web.config.tmp
+/web.config.txt
+/web.Debug.config
+/web.rar
+/web.Release.config
+/web.sql
+/web.tar
+/web.tar.bz2
+/web.tar.gz
+/web.tgz
+/web.xml
+/web.zip
+/web/phpMyAdmin/
+/web/phpMyAdmin/scripts/setup.php
+/web/scripts/setup.php
+/webadmin
+/Webadmin
+/WebAdmin
+/webadmin.html
+/webadmin.php
+/webadmin/
+/WebAdmin/
+/webadmin/admin.html
+/webadmin/admin.php
+/webadmin/index.html
+/webadmin/index.php
+/webadmin/login.html
+/webadmin/login.php
+/Webalizer/
+/webdav/
+/webdav/index.html
+/webdav/servlet/webdav/
+/webdb/
+/webgrind
+/webmail/
+/webmail/src/configtest.php
+/webmaster
+/webmaster.php
+/webmaster/
+/webmin/
+/WebResource.axd?d=LER8t9aS
+/WebSer~1
+/WebService
+/WebServiceServlet
+/WebServicesSamples/docs/
+/WebShell.cgi
+/website.git
+/WebSphere
+/WebSphereBank
+/WebSphereBank/
+/WebSphereBank/docs/
+/WebSphereBankDeposit
+/WebSphereBankDeposit/
+/WebSphereBankDepositservlet
+/WebSphereBankDepositservlet/
+/WebSphereBankservlet
+/WebSphereBankservlet/
+/WebSphereSamples
+/WebSphereSamples.Configuration.config
+/WebSphereSamples/
+/WebSphereSamples/SingleSamples/AccountAndTransfer/create.html
+/WebSphereSamples/SingleSamples/Increment/increment.html
+/WebSphereSamples/YourCo/main.html
+/websql/
+/webstat/
+/webstats.html
+/webstats/
+/wenzhang
+/whmcs.php
+/whmcs/downloads/dz.php
+/wiki
+/wiki/
+/Wishlist.1.marketing.adobe.com.txt
+/wizmysqladmin/
+/WLDummyInitJVMIDs
+/Wordpress/
+/WordPress/
+/wp
+/WP
+/wp-admin
+/wp-admin/
+/wp-admin/c99.php
+/wp-admin/install.php
+/wp-admin/setup-config.php
+/wp-app.log
+/wp-config.1.marketing.adobe.com.txt
+/wp-config.inc
+/wp-config.old
+/wp-config.php.bak
+/wp-config.php.dist
+/wp-config.php.inc
+/wp-config.php.old
+/wp-config.php.save
+/wp-config.php.swp
+/wp-config.php.txt
+/wp-config.php.zip
+/wp-config.php~
+/wp-content
+/wp-includes
+/wp-includes/
+/wp-includes/rss-functions.php
+/wp-login
+/wp-login.1.marketing.adobe.com.txt
+/wp-login.php
+/wp-login/
+/wp-register
+/wp-register.php
+/wp-rss2
+/wp.php
+/wp.rar/
+/wp.zip
+/wpad.dat
+/WS_FTP
+/ws_ftp.ini
+/WS_FTP.ini
+/WS_FTP.log
+/WS_FTP.LOG
+/WS_FTP/
+/WS_FTP/Sites/ws_ftp.ini
+/ws.php
+/wsadmin.traceout
+/wsadmin.valout
+/wsadminListener.out
+/wso.php
+/WSO.php
+/wso2.5.1.php
+/wso2.php
+/WSsamples
+/wstats
+/wvdial.conf
+/www-error.log
+/www.rar
+/www.sql
+/www.tar
+/www.tar.gz
+/www.tgz
+/www.zip
+/wwwboard/passwd.txt
+/wwwroot.7z
+/wwwroot.rar
+/wwwroot.sql
+/wwwroot.tar
+/wwwroot.tar.bz2
+/wwwroot.tar.gz
+/wwwroot.tgz
+/wwwroot.zip
+/wwwstats.htm
+/x.php
+/xampp/phpmyadmin/
+/xampp/phpmyadmin/scripts/setup.php
+/xd.php
+/xferlog
+/xlogin/
+/xls/
+/xml/_common.xml
+/xml/common.xml
+/xmlrpc_server.php
+/xmlrpc.php
+/xphperrors.log
+/xphpMyAdmin/
+/xsl/
+/xsl/_common.xsl
+/xsl/common.xsl
+/xsql/lib/XSQLConfig.xml
+/XSQLConfig.xml
+/yaml_cron.log
+/yaml.log
+/yonetici
+/yonetici.html
+/yonetici.php
+/yonetim
+/yonetim.html
+/yonetim.php
+/yum.log
+/zabbix/
+/zebra.conf
+/zehir.php
+/zeroclipboard.swf
+/zf_backend.php
+/zimbra
```

Comment:
Wondering: is that the zimbra web client? Where did you get this large list of URLs to block?

Reply:
An amateur security researcher kindly provided us with a list of URLs. Until they hit the activation limit, the script aborted and they went back to their other toys. https://cq-dev.slack.com/archives/CGG9B79BK/p1596310680339300